Library/Formula/postgresql.rb

```diff
@@ -45,6 +45,10 @@ class Postgresql < Formula
   def install
     ENV.libxml2 if MacOS.version >= :snow_leopard
 
+    ENV.prepend "LDFLAGS", "-L#{Formula['openssl'].opt_lib}"
```

Comment:
The double quotes inside double quotes works without dancing because of the `[]` surrounding brackets.